.circleci/config.yml

```diff
@@ -161,48 +165,87 @@ workflows:
           image_file: buildpacks-18.tar
-          builder_toml: builder.toml
+          builder_toml: builder-18.toml
           requires:
             - create-18-run-image
             - create-18-build-image
+      - create-pack-builder:
+          name: create-service-builder-20
+          image_tag: heroku/buildpacks:20
+          image_file: buildpacks-20.tar
+          builder_toml: builder-20.toml
+          requires:
+            - create-20-run-image
+            - create-20-build-image
+      - test-getting-started-guide:
+          language: go
+          stack-tag: "18"
+          name: test-go-18
+          requires:
+            - create-service-builder-18
       - test-getting-started-guide:
           language: go
-          name: test-go
+          stack-tag: "20"
+          name: test-go-20
+          requires:
+            - create-service-builder-20
+      - test-getting-started-guide:
+          language: java
+          stack-tag: "18"
+          name: test-java-18
           requires:
-            - create-service-builder
+            - create-service-builder-18
       - test-getting-started-guide:
           language: java
-          name: test-java
+          stack-tag: "20"
+          name: test-java-20
           requires:
-            - create-service-builder
+            - create-service-builder-20
       - test-getting-started-guide:
           language: node-js
-          name: test-node-js
+          stack-tag: "18"
+          name: test-node-js-18
           requires:
-            - create-service-builder
+            - create-service-builder-18
       - test-getting-started-guide:
           language: typescript
-          name: test-typescript
+          stack-tag: "18"
+          name: test-typescript-18
           requires:
-            - create-service-builder
+            - create-service-builder-18
       - test-getting-started-guide:
           language: php
-          name: test-php
+          stack-tag: "18"
+          name: test-php-18
           requires:
-            - create-service-builder
+            - create-service-builder-18
+      - test-getting-started-guide:
+          language: php
+          stack-tag: "20"
+          name: test-php-20
+          requires:
+            - create-service-builder-20
       - test-getting-started-guide:
           language: Python
-          name: test-python
+          stack-tag: "18"
+          name: test-python-18
           requires:
-            - create-service-builder
+            - create-service-builder-18
+      - test-getting-started-guide:
+          language: Python
+          stack-tag: "20"
+          name: test-python-20
+          requires:
+            - create-service-builder-20
       - test-getting-started-guide:
           language: Ruby
-          name: test-ruby
+          stack-tag: "18"
+          name: test-ruby-18
           requires:
-            - create-service-builder
+            - create-service-builder-18
```

Comment:
Ruby's CNB now supports Heroku-20 (https://github.com/heroku/heroku-buildpack-ruby/releases/tag/v222), so we can add a 20 variant here :-)

Reply:
fixed in 643440a